apps/authoring-services/src/main/java/com/infosys/lexauthoringservices/controller/CompetencyController.java

```diff
@@ -0,0 +1,35 @@
+package com.infosys.lexauthoringservices.controller;
+
+import com.infosys.lexauthoringservices.model.Competency;
+import com.infosys.lexauthoringservices.model.CompetencyWebHookResponse;
+import com.infosys.lexauthoringservices.service.CompetencyService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.HashMap;
+
+@RestController
+@RequestMapping("/action/competency")
+public class CompetencyController {
+
+    @Autowired
+    private CompetencyService competencyService;
+
+    @PostMapping(value = "/update", consumes = MediaType.APPLICATION_JSON_VALUE, produces = {MediaType.APPLICATION_JSON_VALUE})
+    public ResponseEntity<CompetencyWebHookResponse> updateCompetencyData(@RequestHeader("rootOrg") String rootOrg, @Valid @RequestBody Competency competency) {
+        competency.setRootOrg(rootOrg);
+        competencyService.updateCompetencyData(competency);
+        CompetencyWebHookResponse response = new CompetencyWebHookResponse();
+        HashMap<String, Object> statusInfo = new HashMap<>();
+        statusInfo.put("statusCode", HttpStatus.ACCEPTED.value());
```

Comment:
Can we make these strings as constant... just check they already available or not...

Reply:
Made the string as constant